EDITORIAL_GUIDELINES.md

```diff
@@ -41,28 +39,44 @@ These guidelines can be modified through a Pull Request (PR), which the members
         <td><strong>Column &amp; Attribute IDs:</strong></td>
         <td>
            <strong>Columns IDs</strong>:<br>
-            - PricingQuantity</strong><br> 
-            - PricingUnit</strong><br> 
-            - ProviderName</strong> <br><br>
+            - PricingQuantity<br> 
+            - PricingUnit<br> 
+            - ProviderName<br><br>
           <strong>Attributes IDs</strong>:<br>
             - CurrencyCodeFormat <br> 
             - DateTimeFormat <br>
         </td>
         <td>
-          <strong>Columns IDs:</strong>:<br>
+          <strong>Columns IDs:</strong><br>
           &nbsp;&nbsp; PricingQuantity <br>
-          &nbsp;&nbsp; PricingUnit</strong><br>
-          &nbsp;&nbsp; ProviderName</strong> <br><br>
+          &nbsp;&nbsp; PricingUnit<br>
+          &nbsp;&nbsp; ProviderName <br><br>
           <strong>Attributes IDs:</strong> </br>
           &nbsp;&nbsp; CurrencyCodeFormat </br>
           &nbsp;&nbsp; DateTimeFormat <br>
         </td>
         <td>
-           - Use PascalCamel case (the first letter of every word, is capitalized)
+           - Use PascalCamel case (the first letter of every word, is capitalized)<br>
            - Normal text without bold or italics.<br>
            - The first occurrence in a section is linked to the section.
         </td>
     </tr>
+    <tr>
+        <td><strong>Column Values:</strong></td>
+        <td>
+            - "Usage"<br>
+            - "Tax"<br>
+            - "TB"<br>
+        </td>
+        <td>
+            This column:<br>
+            &nbsp;&nbsp; * MUST be null when ChargeCategory is "Tax" ...
+        </td>
+        <td>
+            - Enclosed in double quotation marks<br>
+            - Normal text without bold or italics
+        </td>
+    </tr>
```

Comment:
**Current status**: 
While we have consistently formatted normalized dimensions values as normal text without bold or italics, enclosed in double quotation marks, for metrics and non-normalized dimensions values, we've been using various formatting., such as:

- "TB"
- `Hours`
- Compute Usage

**Note/Question:** We internally use the terms normalized and non-normalized dimensions, but we haven't defined them. Should we?

---

OPTION B:

```suggestion
    <tr>
        <td><strong>Normalized Dimension Values:</strong></td>
        <td>
            - "Usage"<br>
            - "Tax"<br>
        </td>
        <td>
            This column:<br>
            &nbsp;&nbsp; * MUST be null when ChargeCategory is "Tax" ...
        </td>
        <td>
            - Enclosed in double quotation marks<br>
            - Normal text without bold or italics
        </td>
    </tr>
    <tr>
        <td><strong>Metric &amp; Non-normalized Dimension Values:</strong></td>
        <td>
            - `Hours`<br>
            - `TB`<br>
            - `Compute Usage`
        </td>
        <td>
            Common examples include the number of hours for compute appliance runtime (e.g. `Hours`) ...
        </td>
        <td>
            - Monospaced text, enclosed in backticks, without bold or italics<br>
        </td>
    </tr>
```

Reply:
The group agreed to go with option A.